SettingsList.py

```diff
@@ -2249,19 +2249,35 @@ def __init__(self, name, gui_text, min, max, default, step=1,
         gui_params     = {
             'randomize_key': 'randomize_settings',
         },
-    ), 
-    Checkbutton(
+    ),
+    Combobox(
         name           = 'mix_entrance_pools',
         gui_text       = 'Mix Entrance Pools',
+        default        = 'off',
+        choices        = {
+            'off':       'Off',
+            'non-ow':    'Non-Overworld',
```

Comment:
I suggest "Indoor Entrances" for the name of the intermediate option.
```suggestion
            'indoor':    'Indoor Entrances',
```

Reply:
Agree, much clearer this way